model/aspnetcore/metrics.yaml

```diff
@@ -114,3 +125,166 @@ groups:
     attributes:
       - ref: aspnetcore.rate_limiting.result
         requirement_level: required
+
+  # blazor components
+  - id: metric.aspnetcore.components.navigation
+    type: metric
+    metric_name: aspnetcore.components.navigation
+    stability: development
+    brief: Total number of route changes.
+    instrument: counter
+    unit: "{route}"
+    note: |
+      Meter name: `Microsoft.AspNetCore.Components`; Added in: ASP.NET Core 10.0
+    extends: aspnetcore.components.metrics.attributes
+    attributes:
+      - ref: aspnetcore.components.type
+        requirement_level: required
+      - ref: aspnetcore.components.route
+        requirement_level: required
+
+  - id: metric.aspnetcore.components.event.duration
+    type: metric
+    metric_name: aspnetcore.components.event.duration
+    stability: development
+    brief: Duration of processing browser event.
```

Comment:
event in the context of telemetry is very special (usually something that does not have any duration). Could we consider a different noun here? e.g. `aspnetcore.component.operation | call.duration`

Reply:
Do you mean [this](https://opentelemetry.io/docs/specs/semconv/general/events/) ?

They speak about button click. This is actually button click handler duration.

But it's not timestamp. In the Blazor server,we get this event only after it arrived over wire.
Processing the handler it has (business logic and a) duration.

Reply:
OTEL [expects](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.44.0/specification/logs/data-model.md#field-eventname) `eventname`

Blazor internally refers to it by `attributeName` which is the binding between browser event and blazor handler.

We discuss it here https://github.com/dotnet/aspnetcore/pull/61609#discussion_r2069650759

Reply:
event_handler now